asyncapi/google.drive/service_types.bal

```diff
@@ -14,7 +14,9 @@
 // specific language governing permissions and limitations
 // under the License.
 
-
+# Triggers when a new event related to Google Drive event is received.
```

Comment:
Google Drive event ->. Google Drive

Reply:
Resolved all, Thanks a lot for the suggestions.